compose/mpp/demo/project.template.yml

```diff
@@ -14,7 +15,9 @@ targets:
     platform: iOS
     deploymentTarget: "12.0"
     prebuildScripts:
-      - script: cd "$SRCROOT" && OUT_DIR=`pwd`/out  JAVA_TOOLS_JAR=/Users/jetbrains/jetpack_compose_ui/compose-jb/compose/scripts/../external/tools.jar  ALLOW_PUBLIC_REPOS=1  ANDROIDX_PROJECTS=compose ../../../gradlew  -i  -Pandroidx.compose.multiplatformEnabled=true  --no-daemon -Pkotlin.compiler.execution.strategy="in-process" -p . packForXCode
+      - script: |
+          export JAVA_HOME=%@JAVA_HOME@%
+          cd "$SRCROOT" && OUT_DIR=`pwd`/out  JAVA_TOOLS_JAR=/Users/jetbrains/jetpack_compose_ui/compose-jb/compose/scripts/../external/tools.jar  ALLOW_PUBLIC_REPOS=1  ANDROIDX_PROJECTS=compose ../../../gradlew  -i  -Pandroidx.compose.multiplatformEnabled=true  --no-daemon -Pkotlin.compiler.execution.strategy="in-process" -p . packForXCode
```

Comment:
Why do we need `/../` if it's absolute path?

Reply:
Better to address this question to @dima-avdeev-jb, I didn't change this code.

Reply:
I think we don't need JAVA_TOOLS_JAR anymore